opencensus/stats/internal/view_data_impl.cc

```diff
@@ -65,6 +65,7 @@ ViewDataImpl::ViewDataImpl(absl::Time start_time,
       aggregation_window_(descriptor.aggregation_window_),
       type_(TypeForDescriptor(descriptor)),
       start_time_(start_time) {
+  new (&start_times_) DataMap<absl::Time>();
```

Comment:
This shouldn't be needed, the placement news below are because of the union { ... } funny-business, but start_times_ is just a regular member.

Reply:
Removed all placement news